src/App/Fossa/Init/.fossa.yml

```diff
@@ -0,0 +1,196 @@
+#   #   Otherwise, they will be silently ignored.
+#   team: cli-team
+#
+#   # Name of the policy in your FOSSA organization to associate with this project.
+#   # NOTE:
+#   #   Can only be set when creating a project (running fossa analyze for the first time),
+#   #   Otherwise, they will be silently ignored.
+#   policy: custom-cli-policy
+#
+#   # An external link that will appear in the FOSSA UI for this specific project.
+#   # NOTE:
+#   #   Can only be set when creating a project (running fossa analyze for the first time),
+#   #   Otherwise, they will be silently ignored.
+#   link: fossa.com
+#
+#   # The URL of your project that will appear in FOSSA.
+#   # Intended to be the URL to the repository of this project.
+#   url: github.com/fossas/fossa-clif
+#
+#   # Jira Project Key to associate with your project for improved issue triage.
+#   # See documentation for details: https://docs.fossa.com/docs/atlassian-jira
+#   #
+#   # NOTE:
+#   #   Can only be set when creating a project (running fossa analyze for the first time),
+#   #   Otherwise, they will be silently ignored.
+#   jiraProjectKey: jira-key
+#
+#   # The 'name' and 'release' of the release group's release to add your project to in the FOSSA dashboard.
+#   # See documentation for details: https://docs.fossa.com/docs/release-groups
+#   #
+#   # NOTE:
+#   #   If you choose to associate a project with a release group, you must supply both name and release.
+#   releaseGroup:
+#     name: release-group-name
+#     release: 123-release-candidate
+#
+#   # Allows you to add labels to projects for classification in the FOSSA UI.
+#   # See documentation for details: https://docs.fossa.com/docs/projects-ui-whats-new#labels
+#   labels:
+#     - project-label-1
+#     - test-project
+#
+#
+# # Revision associated with project scan.
+# revision:
+#   # Used to identify a specific scan for a project.
+#   # Default:
+#   #   - Git: Commit hash of the HEAD branch (inferred from current HEAD state in the .git directory)
+#   #   - SVN: "Revision" obtained using 'svn info' command.
+#   #   - No VCS: Unix timestamp.
+#   commit: "12345"
+#
+#   # Optional setting used for organizing project revisions in the FOSSA UI.
+#   # Default:
+#   #   - Git: Current branch from .git/config file.
+#   #   - SVN: "URL" or "Repository Root" fields using `svn info` command.
+#   #   - No VCS: Empty.
+#   branch: master
+#
+#
+# # Vendored Dependencies Configuration
+# # See documentation for details: https://github.com/fossas/fossa-cli/blob/master/docs/features/vendored-dependencies.md
+# vendoredDependencies:
+#   # If true, forces a re-scan of all vendored dependencies on every run. Default: false
+#   forceRescans: false
+# 
+#   # Determines whether vendored dependencies are scanned using "Archive Upload" or "CLI License Scan" method.
+#   # Possible values: ArchiveUpload, CLILicenseScan. Default: CLI License Scan (organization default)
```

Comment:
[nit]: the default should be `CLILicenseScan`, without the spaces.

Also, I think we should expand more on "organization default". Maybe something like this (stolen from the documentation):

> The default is usually "CLILicenseScan", but your organization may have opted to default to "ArchiveUpload".